### ISSUE TITLE
java.lang.IllegalStateException - android.graphics.Canvas.__constructor__

### ISSUE THREAD

**Issue by chrisbanes** [robolectric/robolectric#4072]
### Description
I've just upgraded to 4.0.1 and I'm now hitting an IllegalStateException in Canvas. Seems to be when using a fragment transition.

```
[Robolectric] app.tivi.HomeActivityNavigationTests.testBottomNavigationLibraryClick: sdk=28; resources=binary
Called loadFromPath(/system/framework/framework-res.apk, true); mode=binary sdk=28

java.lang.IllegalStateException
	at android.graphics.Canvas.__constructor__(Canvas.java:128)
	at android.graphics.Canvas.<init>(Canvas.java)
	at android.view.RecordingCanvas.<init>(RecordingCanvas.java)
	at android.view.DisplayListCanvas.<init>(DisplayListCanvas.java)
	at android.view.DisplayListCanvas.obtain(DisplayListCanvas.java:55)
	at android.view.RenderNode.start(RenderNode.java:218)
	at android.graphics.Bitmap.createBitmap(Bitmap.java:1243)
	at android.graphics.Bitmap.createBitmap(Bitmap.java:1205)
	at androidx.transition.TransitionUtils.createViewBitmap(TransitionUtils.java:127)
	at androidx.transition.TransitionUtils.copyViewImage(TransitionUtils.java:64)
	at androidx.transition.Visibility.onDisappear(Visibility.java:363)
	at androidx.transition.Visibility.createAnimator(Visibility.java:253)
	at androidx.transition.Transition.createAnimators(Transition.java:744)
	at androidx.transition.TransitionSet.createAnimators(TransitionSet.java:470)
	at androidx.transition.TransitionSet.createAnimators(TransitionSet.java:470)
	at androidx.transition.Transition.playTransition(Transition.java:1817)
	at androidx.transition.TransitionManager$MultiListener.onPreDraw(TransitionManager.java:295)
	at android.view.ViewTreeObserver.dispatchOnPreDraw(ViewTreeObserver.java:977)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2474)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1460)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:7183)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at org.robolectric.shadows.ShadowMessageQueue.dispatchMessage(ShadowMessageQueue.java:144)
	at org.robolectric.shadows.ShadowMessageQueue.access$200(ShadowMessageQueue.java:35)
	at org.robolectric.shadows.ShadowMessageQueue$1.run(ShadowMessageQueue.java:123)
	at org.robolectric.util.Scheduler$ScheduledRunnable.run(Scheduler.java:386)
	at org.robolectric.util.Scheduler.runOneTask(Scheduler.java:278)
	at org.robolectric.util.Scheduler.advanceTo(Scheduler.java:260)
	at org.robolectric.util.Scheduler.advanceBy(Scheduler.java:243)
	at org.robolectric.util.Scheduler.advanceBy(Scheduler.java:233)
	at org.robolectric.util.Scheduler.runOrQueueRunnable(Scheduler.java:353)
	at org.robolectric.util.Scheduler.postDelayed(Scheduler.java:162)
	at org.robolectric.util.Scheduler.postDelayed(Scheduler.java:151)
	at org.robolectric.shadows.ShadowMessageQueue.enqueueMessage(ShadowMessageQueue.java:130)
	at android.os.MessageQueue.enqueueMessage(MessageQueue.java)
	at android.os.Handler.enqueueMessage(Handler.java:745)
	at android.os.Handler.sendMessageAtTime(Handler.java:697)
	at android.os.Handler.sendMessageDelayed(Handler.java:667)
	at android.os.Handler.post(Handler.java:395)
	at androidx.test.espresso.base.BaseLayerModule$1.execute(BaseLayerModule.java:92)
	at androidx.test.espresso.ViewInteraction.runSynchronouslyOnUiThread(ViewInteraction.java:303)
	at androidx.test.espresso.ViewInteraction.desugaredPerform(ViewInteraction.java:161)
	at androidx.test.espresso.ViewInteraction.perform(ViewInteraction.java:114)
	at app.tivi.HomeActivityNavigationTests.testBottomNavigationLibraryClick(HomeActivityNavigationTests.kt:51)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
```

### Robolectric & Android Version
Android SDK v28
Robolectric 4.0.1

### Link to a public git repo demonstrating the problem:
https://github.com/chrisbanes/tivi/blob/cb/robolectric-app-tests/app/src/test/java/app/tivi/HomeActivityNavigationTests.kt
(note the branch)

**Comment by xian** [comment]
Because of missing impl for `DisplayListCanvas.nCreateDisplayListCanvas()`.

**Comment by jongerrish** [comment]
I have a fix for this and it should be in 4.0.2
